### ISSUE TITLE
App crashes when image is caching 

### ISSUE THREAD

**Issue by owlstack** [luberda-molinet/FFImageLoading#345]
On Xamarin.Forms app, when the app is first starting up and the images are being loaded for caching, the app crashes. This happens on the prelease of the latest app. 

Later tonight, I'll see if I can get the exact exception but I think it was a null reference.

**Comment by daniel-luberda** [comment]
StackTrace would be great. I tried to reproduce it with: 

```C#
ImageService.Instance.LoadCompiledResource("loading.png").Preload();
ImageService.Instance.LoadUrl("http://loremflickr.com/600/600/nature?filename=simple.jpg").DownloadOnly();
```

but didn't get any exceptions.

**Comment by daniel-luberda** [comment]
Found the issue. Just call `ImageService.Instance.Initialize()` before using FFImageLoading. 

I'll fix this.

**Comment by daniel-luberda** [comment]
It's fixed, just download `2.2.0-pre-171` package. The exception was thrown when no custom logger was provided. Sorry for that.
